zkevm-circuits/src/evm_circuit/execution/blockhash.rs

```diff
@@ -36,38 +35,39 @@ impl<F: Field> ExecutionGadget<F> for BlockHashGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::BLOCKHASH;
 
     fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
-        let block_number = cb.query_word_rlc();
-        cb.stack_pop(block_number.expr());
-
         let current_block_number = cb.query_cell();
         cb.block_lookup(
             BlockContextFieldTag::Number.expr(),
             None,
             current_block_number.expr(),
         );
 
-        let block_lt = LtGadget::construct(
-            cb,
-            from_bytes::expr(&block_number.cells),
-            current_block_number.expr(),
-        );
+        let block_number = WordByteCapGadget::construct(cb, current_block_number.expr());
```

Comment:
We're using `WordByteCapGadget` for `block_number`, but I don't see any place where we check `block_number.overflow` or `block_number.within_range`. So the internal `IsZero` gadget seems to be unused?